### ISSUE TITLE
Instructions of how to validate the GnuPG signature of the source code are wrong

### ISSUE THREAD

**Issue by leo-bogert** [etotheipi/BitcoinArmory#91]
I'm talking about this:
https://bitcoinarmory.com/building-armory-from-source/

It tells the user to use "git tag -v" to verify the signature. This is good, but it won't protect against commits which happened AFTER the tag: They will still be checked out even though they are not signed.

You should tell the user to use "git checkout" to obtain the very same tag which they validated using "git tag -v"

**Comment by etotheipi** [comment]
I apologize: we do not typically track the github issues (and apparently I do not get notifications for them anymore).  We typically handle these things through the support channel, though we may not necessarily be able to address everything that comes through there.

I have updated the webage with more information about building from source on every supported OS, and beefed up instructions for verifying source code authenticity.  

Since my original message in this thread, we have made a fairly elaborate build script.  It handles the complex web of making the offline bundles, creating hashes, signing packages, creating announcement files, *and creating and signing a git tag that can be pushed.*  You'll notice that every release since v0.90 has a valid signed tag.

https://github.com/etotheipi/BitcoinArmory/tree/master/release_scripts

Our build process is not yet deterministic.  We are actively integrating Gitian into our build process, though it is a fairly complex process given the complexities of building releases on each OS.  We are working on Linux first, and then will try to get all OSes building under a common build system compatible with Gitian.
